extensions/default/src/ViewerLayout/constants/panels.ts

```diff
@@ -0,0 +1,30 @@
+const panelGroupDefination = {
+  groupId: 'viewerLayoutResizablePanelGroup',
+  shared: {
+    expandedInsideBorderSize: 4,
+    collapsedInsideBorderSize: 4,
+    collapsedOutsideBorderSize: 8,
+    collapsedWidth: 25,
+  },
+  left: {
+    // id
+    panelId: 'viewerLayoutResizableLeftPanel',
+    // expanded width
+    expandedDefaultWidth: 282,
+    // expanded width + expanded inside border
+    expandedDefaultOffsetWidth: 145 + 4,
```

Comment:
I don't understand why we have an `expandedDefaultOffsetWidth` and an `initialExpandedDefaultOffsetWidth`. Furthermore why is `expandedDefaultOffsetWidth` so small?

Reply:
Ok. I think I understand. We have an initial expanded width and then a minimum width. Right? Let's rename them. Maybe something like...

`expandedDefaultWidth` rename to `initialExpandedWidth`
`expandedDefaultOffsetWidth` rename to `minimumExpandedOffsetWidth`
`initialExpandedDefaultOffsetWidth` rename to `initialExpandedOffsetWidth`

To me default and initial kinda mean the same thing. So initial and default in the same variable name was driving me crazy. 🤣

Reply:
Haha i agree, the naming was confusing, I'll fix it now.